Source/AwsCommonRuntimeKit/http/HttpRequest.swift

```diff
@@ -2,18 +2,15 @@
 //  SPDX-License-Identifier: Apache-2.0.
 import AwsCHttp
 import AwsCIo
-
+import AwsCCommon
 public final class HttpRequest: HttpMessage {
```

Comment:
utterly trivial: it's nice to have a line of whitespace between the imports and class declaration